readme.md

```diff
@@ -92,19 +109,20 @@ artists_new GET    /artists/new(.:format)      artists#new
 ```
 Rake routes allows us at any time to view the current routes of this app and see the controller actions mapped to each route.
 
-For example, let's look at the request/response life cycle of a get request hitting the artists index action.
+For example, let's look at the request/response life cycle of a get request hitting the `artists` `index` action.
```

Comment:
I think it would be worthwhile to have the MVC diagram drawn on the whiteboard, and CONSTANTLY reference it as you go through the request/response cycle